src/shared/popups/gokb-edit-platform-popup/gokb-edit-platform-popup.vue

```diff
@@ -0,0 +1,230 @@
+                <gokb-text-field
+                  v-model="localAction"
+                  :label="$t('component.platform.edit')"
+                  disabled
+                />
+              </v-col>
+            </v-row>
+          </div>
+          <div v-else>
+            <v-row dense>
+              <v-col cols="6">
+                <gokb-text-field
+                  v-model="localAction"
+                  :label="$t('component.platform.create')"
+                  disabled
+                />
+              </v-col>
+            </v-row>
+          </div>
+        </template>
+      </v-col>
+    </v-row>
+    <v-row
+      dense
+      class="mt-4"
+    >
+      <v-col>
+        <gokb-title-field
+          v-model="platform.title"
+          :label="$tc('component.platform.label')"
+          required
+          dense
+          return-object
+        />
+      </v-col>
+    </v-row>
+    <v-row dense>
+      <v-col>
+        <gokb-url-field
+          ref="platformUrl"
+          v-model="platform.url"
+          :label="$tc('component.tipp.url.label')"
+          required
+          dense
+        />
+      </v-col>
+    </v-row>
+
+    <template #buttons>
+      <v-spacer />
+      <gokb-button
+        @click="close"
+      >
+        {{ $t('btn.cancel') }}
+      </gokb-button>
+      <gokb-button
+        v-if="!isReadonly"
+        :disabled="!isValid"
+        default
+      >
+        {{ isEdit ? $t('btn.update') : $t('btn.create') }}
+      </gokb-button>
+    </template>
+  </gokb-dialog>
+</template>
+
+<script>
+  import BaseComponent from '@/shared/components/base-component'
+  import accountModel from '@/shared/models/account-model'
+  import platformServices from '@/shared/services/platform-services'
+  import 'vue-json-pretty/lib/styles.css'
+
+  export default {
+    name: 'GokbEditPlatformPopup',
+    extends: BaseComponent,
+    props: {
+      selected: {
+        type: Object,
+        required: false,
+        default: undefined
+      },
+      readonly: {
+        type: Boolean,
+        required: false,
+        default: false
+      },
+      component: {
+        type: Object,
+        required: false,
+        default: undefined
+      }
+    },
+    data () {
+      return {
+        selectedItem: undefined,
+        successMsg: undefined,
+        errorMsg: undefined,
+        updateUrl: undefined,
+        platform: {
+          label: undefined,
+          url: undefined
+        },
+        items: []
+      }
+    },
+    computed: {
+      localValue: {
+        get () {
+          return this.value || true
+        },
+        set (localValue) {
+          this.$emit('input', localValue)
+        }
+      },
+      isReadonly () {
+        return !accountModel.loggedIn() || !accountModel.hasRole('ROLE_EDITOR') || (this.isEdit && !this.updateUrl)
+      },
+      isEdit () {
+        return !!this.id
+      },
+      isValid () {
+        return !!this.platformItem.component
```

Comment:
The data object used in the template is `platform`. Validity in this case would probably require `!!this.platform.name && !!this.platform.url`.